app/src/main/java/io/homeassistant/companion/android/sensors/SensorReceiver.kt

```diff
@@ -103,6 +98,10 @@ class SensorReceiver : BroadcastReceiver() {
         }
     }
 
+    private fun isSensorEnabled(context: Context, id: String): Boolean? {
+        return AppDatabase.getInstance(context).sensorDao().get(id)?.enabled
+    }
```

Comment:
If you make this return a `Boolean` not a `Boolean?` you won't need those dangerous `!!` when calling the function.
```suggestion
    private fun isSensorEnabled(context: Context, id: String): Boolean {
        return AppDatabase.getInstance(context).sensorDao().get(id)?.enabled == true
    }
```

Reply:
Corrected as prposed